tests/components/cert_expiry/helpers.py

```diff
@@ -0,0 +1,8 @@
+"""Helpers for Cert Expiry tests."""
+from datetime import datetime, timedelta
+
+
+def make_timestamp(days):
+    """Create timestamp object for requested days in future."""
+    delta = timedelta(days=days, minutes=1)
+    return datetime.now() + delta
```

Comment:
Done use changing values for tests. Please patch with exact fixed values.

Reply:
Can you clarify why this is bad practice? The purpose of this sensor is to work with timestamps that are X days in the future.

Reply:
I can cause flaky tests. For example with daylight savings changes right during test, some ntp time update cause time to change during test... And so on. you don't want anything non deterministic in tests.

I would patch dt.now() or use a freezetime fixture to control the exact base time in your tests. Then you don't need future timestamp data. You can test against specific values.